app/assets/javascripts/index/directions/graphhopper.js

```diff
@@ -79,5 +80,6 @@ function GraphHopperEngine(id, vehicleType) {
   };
 }
 
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_bicycle", "bike"), true);
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_foot", "foot"), true);
+OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_car", "car", true), true);
```

Comment:
Well I would have thought that if needed to be disabled for the car profile then that should be something the server did automatically as part of the profile rather than forcing the client to deal with implementation details...